### ISSUE TITLE
finish event not triggered when resizing in select mode

### ISSUE THREAD

**Issue by Thunder7991** [JamesLMilner/terra-draw#209]
Hello @JamesLMilner , during my use, the polygon can trigger the finish event after editing is completed. Rectangle and circle not triggered

```typescript
let draw = new TerraDraw({
   ...
})

 draw .on('finish', (e: string) => {
   //TODO
  });
```

After the drag and drop is completed, trigger the finish event

**Comment by JamesLMilner** [comment]
Good catch, should be an easy fix!

**Comment by JamesLMilner** [comment]
This should be available in 0.0.1-alpha.60
